File: bnc/src/main/java/org/sqlbuilder/bnc/BncWordResolver.java
package org.sqlbuilder.bnc;

import org.sqlbuilder.common.DeSerialize;
import org.sqlbuilder.common.Resolver;

import java.io.File;
import java.io.IOException;

public class BncWordResolver extends Resolver<String,Integer>
{
	public BncWordResolver(final String ser) throws IOException, ClassNotFoundException
	{
		super(DeSerialize.deserialize(new File(ser)));
	}
}

File: bnc/src/main/java/org/sqlbuilder/bnc/BncWordResolver.kt
package org.sqlbuilder.bnc

import org.sqlbuilder.common.DeSerialize.deserialize
import org.sqlbuilder.common.Resolver
import java.io.File

typealias BncWordResolvable = String
typealias BncWordResolved = Int

class BncWordResolver(ser: String) : Resolver<BncWordResolvable, BncWordResolved>(deserialize<Map<BncWordResolvable, BncWordResolved>>(File(ser)))
